auto_nag/scripts/regression.py

```diff
@@ -33,6 +44,7 @@ def get_bz_params(self, date):
         reporter_skiplist = ",".join(reporter_skiplist)
 
         params = {
+            "include_fields": ["id", "groups", "summary"],
```

Comment:
"id" and "summary" are added anyway... so no need to have them here.
But if you find that more clear: it's ok for me.

Reply:
I found it clearer this way, I wouldn't know where to document the fact that `id` and `summary` are returned anyway. Is it a specific libmozdata behavior or a Bugzilla itself behavior?

Reply:
It's done here: https://github.com/mozilla/relman-auto-nag/blob/master/auto_nag/bzcleaner.py#L278